turbo.json

```diff
@@ -25,8 +25,7 @@
     "VITE_CLERK_*",
     "EXPO_PUBLIC_CLERK_*",
     "REACT_APP_CLERK_*",
-    "NEXT_PHASE",
-    "E2E_NEXTJS_VERSION"
```

Comment:
This would bust the build cache across tests against diff next versions, which we don't want.